manifests/vhost.pp

```diff
@@ -2362,6 +2362,10 @@
       if 'setenv' in $directory {
         include apache::mod::env
       }
+
+      if 'expires_active' in $directory or 'expires_default' in $directory or 'expires_by_type' in $directory {
+        include apache::mod::expires
```

Comment:
One thing to note: `apache::mod::expires` by default writes down a configuration file (unlike other modules). That could be a behavior difference and break setups. Is that a problem?

Reply:
Don't we have other modules that do that and thus we include them differently?

Reply:
Like
```
::apache::mod { 'expires': }
```

Reply:
That is exactly the point I tried to make and I'm surprised that CI didn't fail on this:
https://github.com/puppetlabs/puppetlabs-apache/blob/6d218cfcbcb1c58cbf77ccd5669524792172c7c0/manifests/default_mods.pp#L63
https://github.com/puppetlabs/puppetlabs-apache/blob/6d218cfcbcb1c58cbf77ccd5669524792172c7c0/manifests/default_mods.pp#L93

It feels ugly, but perhaps we should have some helper class that is only the module definition without the configuration file.